sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/AzureBlobStorageBuilder.java

```diff
@@ -91,7 +91,11 @@ public AzureBlobStorageImpl build() {
         if (this.url != null) {
             client.setUrl(this.url);
         }
-        client.setVersion(this.version);
+        if (this.version != null) {
```

Comment:
We read this from another class, instead of hard coding here? Did you pull the change?